components/expenses/PayExpenseModal.js

```diff
@@ -193,6 +194,12 @@ const PayExpenseModal = ({ onClose, onSubmit, expense, collective, error }) => {
         {error && (
           <MessageBox type="error" withIcon my={3}>
             {error}
+            {error ===
+              'Not enough funds in your existing Paypal preapproval. Please refill your PayPal payment balance.' && (
```

Comment:
Using the message is not a very nice way to detect the error because if the message changes it will break this code. But we don't have an `id` for this error type, so that's the simplest way to do it. Can we do:

```suggestion
            {/** TODO: Add a proper ID/Type to detect this error */}
            {error.startsWith('Not enough funds in your existing Paypal preapproval') && (
```